webapp/src/components/Board/PieceComponent.jsx

```diff
@@ -0,0 +1,157 @@
+import React, { useContext } from 'react';
+import axios from 'axios';
+import BoardContext from './BoardContext';
+
+const PieceComponent = ({
+	skin,
+	color,
+	name,
+	selected,
+	targeted,
+	idle,
+	index,
+	turn,
+	access,
+	from,
+	lobbyID,
+	chess,
+}) => {
+	/**
+	 * Вернет true, если клетка черная
+	 * @param {int} ind 0 - 63
+	 * @returns
+	 */
+	const isBlackCell = (ind) => {
+		const row = Math.floor(ind / 8);
+		const col = ind % 8;
+		return (row % 2 === 0 && col % 2 === 1) || (row % 2 === 1 && col % 2 === 0);
+	};
+	/**
+	 * Преобразует индекс клетки в классическую нотацию
+	 * @param {int} ind 0 - 63
+	 * @returns
+	 */
+	const getCODE = (ind) => {
```

Comment:
Could be written as
`getCode = ind => '${String.fromCharCode(charCode+(ind % 8))}${8-Math.floor(ind / 8)}'`
(<`> symbol instead of <'>)
charCode = 65 // 'A'
charCode = 97 // 'a'